metis-indexing/src/main/java/eu/europeana/indexing/solr/property/QualityAnnotationSolrCreator.java

```diff
@@ -0,0 +1,24 @@
+package eu.europeana.indexing.solr.property;
+
+import eu.europeana.corelib.definitions.edm.entity.QualityAnnotation;
+import eu.europeana.indexing.solr.EdmLabel;
+import org.apache.solr.common.SolrInputDocument;
+
+/**
+ * @author Simon Tzanakis (Simon.Tzanakis@europeana.eu)
+ * @since 2019-06-13
+ */
+public class QualityAnnotationSolrCreator implements PropertySolrCreator<QualityAnnotation> {
+
+  @Override
+  public void addToDocument(SolrInputDocument doc, QualityAnnotation qualityAnnotation) {
+    final EdmLabel tierTypeLabel;
+    if (qualityAnnotation.getAbout().contains("metadataTier")) {
+      tierTypeLabel = EdmLabel.METADATA_TIER;
+    } else {
```

Comment:
A couple of small things:
1. Maybe we should check for '#metadataTier' (with hash) to reduce the (already remote) chance that a tier will be added with a name that also contains metadataTier.
2. I think we should also add a condition to the else block that checks the about. Maybe a different value appears for some reason, in which case we should probably skip this value.